Assets/Plugins/Clean/Editor/ComponentProperties.cs

```diff
@@ -0,0 +1,213 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Plugins.Clean.Editor
+{
+    public static class ComponentProperties
+    {
+        public class SelectableObject
+        {
+            private readonly AnimationTriggers animationTriggers;
```

Comment:
FYI: For naming conventions, here's the one I saw being used the most https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/

Unity does use its own naming convention (more akin to what you're using).

My opinion is that it's just a convention and people use what they like, but once committed on a convention, I do like to use the same convention in the whole project. (Not entirely what Artstorm is doing though)